relay-replays/src/lib.rs

```diff
@@ -147,7 +147,7 @@ struct VersionedMeta {
 
 #[derive(Debug, Default, Deserialize, Serialize)]
 struct User {
-    id: Option<String>,
+    id: Option<Value>,
```

Comment:
Just a quick question: is User ID can be anything or is there an agreed upon definition of how the user should looks like?

Reply:
String or number.  Mostly I'm just not concerned with validating it here.  At least not yet. I want to allow anything for now and let the snuba consumer sort it out.

I want to use `relay_general::protocol::user::User` but it was a little difficult to integrate and its something I want to come back to.  If you have any tips for utilizing it let me know.